client/src/templates/Introduction/components/challenges.tsx

```diff
@@ -29,10 +28,6 @@ function Challenges({
   blockTitle
 }: Challenges): JSX.Element {
   const { t } = useTranslation();
-
-  const renderCheckMark = (isCompleted: boolean) =>
-    isCompleted ? <GreenPass /> : <GreenNotCompleted />;
```

Comment:
This is needed to **not** hush the screen reader text.

Reply:
Yeah, didn't notice, I am going to go with the previous PRs then